data/brands/amenity/cafe.json

```diff
@@ -41,6 +41,23 @@
     }
   },
   "items": [
+    {
+      "displayName": "7 Brew",
+      "locationSet": {"include": ["us"]},
+      "matchNames": [
+        "7Brew",
```

Comment:
By the way, you don’t need to worry about case or spaces in matchNames. Clients like iD and Rapid already do a bit of munging to catch similar names. Stuff like “coffee” at the end or spelling out “seven” is useful though.